charts/seed-bootstrap/aggregate-prometheus-rules/fluent-bit.rules.yaml

```diff
@@ -19,7 +19,25 @@ groups:
         description: >
           There are no fluent-bit pods running on seed: {{$externalLabels.seed}}.
           No logs will be collected.
-  
+
+    - alert: FluentBitIdleInputPlugins
+      expr: |
+        sum by (pod) (
+          increase(
+            fluentbit_input_bytes_total{pod=~"fluent-bit.*"}[4m]
+          )
+        ) == 0
+      for: 6h
```

Comment:
Hm, isn't 6 hours too long?

Reply:
I don't know. There is two cases when this alert can be triggered without hitting this bug.
1. Non of the monitored component is logging event a single line for a period of time.
    There was a similar alert which was triggered on dev and staging when there was a seed with one or few clusters(which were in hibernated state) and there was nothing to be logged on one of its node.
    The systemd input plugin should produce some logs after all but the question is for how long we have to wait.
 2. The node is under  high pressure and the input buffer is full. When the output plugging is processing the logs too slow the inputs are blocked until there is some free space in the buffer. In this case we have either a structural defect in the fluent-bit flow or we face network outage or massive Loki down event.
 
 So I don't have any evidence what will be the proper time.
 I am open for proposals.

Reply:
My point is that for 6 hours anything can happen and something else can restart stuck pod and you will not find. Of course, the disadvantage of shorter firing times is that more false positive alerts might be generated.

Personally, I have no hard opinion.